src/actions/index.js

```diff
@@ -10,36 +16,53 @@ export const removeFromCart = cartItem => {
 }
 
 const loadCartItems = () => {
-  return localforage.getItem('cartItems')
+  return localforage.getItem(CART_ITEM_KEY);
 }
 
 const loadCartAddress = () => {
-  return localforage.getItem('cartAddress')
+  return localforage.getItem(CART_ADDRESS_KEY);
 }
 
 const loadUser = () => {
-  return localforage.getItem('user')
+  return localforage.getItem('user');
 }
 
 const loadRestaurant = (client, restaurantId) => {
-  return client.get('/api/restaurants/' + restaurantId)
+  return client.get('/api/restaurants/' + restaurantId);
 }
 
 const loadDeliveryDate = () => {
-  return localforage.getItem('deliveryDate')
+  return localforage.getItem(DELIVERY_DATE_KEY);
+}
+
+const loadLastOrder = () => {
+  return localforage.getItem(ORDER_KEY);
 }
 
 export const initialize = (baseURL, restaurantId) => (dispatch, getState) => {
   localforage.getItem('coopcyle__api_credentials')
     .then(credentials => {
```

Comment:
Maybe you would appreciate that we use the new cool in stage async/await syntax
(https://hackernoon.com/6-reasons-why-javascripts-async-await-blows-promises-away-tutorial-c7ec10518dd9)
... we need just to change the transpiler by using the babel-preset-env or babel-preset-latest...

and then cascade of Promises are not anymore a nightmare for the eyes:).